CMakeLists.txt

```diff
@@ -70,33 +68,196 @@ set(GLFW_BUILD_EXAMPLES OFF CACHE STRING "" FORCE)
 set(GLFW_BUILD_TESTS OFF CACHE STRING "" FORCE)
 set(GLFW_BUILD_DOCS OFF CACHE STRING "" FORCE)
 set(BUILD_BULLET3 OFF CACHE STRING "" FORCE) # Can use bullet2, bullet3 wants to build examples...
-
 add_subdirectory(lib/bullet3)
 
-include_directories(lib/bullet3/src)
-target_link_libraries(${PROJECT_NAME} BulletDynamics BulletCollision LinearMath)
-
-
 # MoltenTempest
 set(BUILD_SHARED_LIBS ${BUILD_SHARED_MOLTEN_TEMPEST})
 add_subdirectory(lib/MoltenTempest/Engine)
 
+
+## OpenGothic
+
+# sources
+# FIXME: recursive CMakeLists.txt instead of global listing
+set(OPENGOTHIC_SOURCES
```

Comment:
>  recursive CMakeLists.txt instead of global listing

OK, but why, not to keep wildcards for now?

Reply:
Alright, will revert for now.